ext/clementine-tagreader/main.cpp

```diff
@@ -60,8 +60,6 @@ int main(int argc, char** argv) {
 
   QSslSocket::addDefaultCaCertificates(
       QSslCertificate::fromPath(":/certs/godaddy-root.pem", QSsl::Pem));
```

Comment:
@hatstand it looks like the godaddy certificate was added to support UbuntuOne. Do you recall?

Reply:
No idea, sorry